spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -275,11 +275,11 @@ class CodeEditor(TextEditBaseWidget):
     lsp_response_signal = Signal(str, dict)
 
     #: Signal to display object information on the Help plugin
-    sig_display_object_info = Signal(str)
+    sig_display_object_info = Signal(object)
 
     #: Signal only used for tests
     # TODO: Remove it!
-    sig_signature_invoked = Signal()
+    sig_signature_invoked = Signal(object)

```

Comment:
Also this one?

Reply:
Same as above, but in this case I need to be able to check on the tests what the reply was 

https://github.com/spyder-ide/spyder/pull/9301/files#diff-d6fbacae4782021d26229f59f1a4a082R101